src/extension/background.js

```diff
@@ -12,9 +12,18 @@ const bgImages = require.context("../images/patterns/", false, /bg-.*\.svg/);
 const siteUrl = process.env.SITE_URL;
 
 const init = () => {
-  browser.browserAction.onClicked.addListener(() =>
-    browser.tabs.create({ url: siteUrl })
-  );
+  browser.browserAction.onClicked.addListener(() => {
+    browser.tabs.query({ currentWindow: true })
+      .then(tabs => {
+        const themerTab = tabs.find(tab => {
+          return tab.title === "Firefox Themer";
+        });
+        if (themerTab)
```

Comment:
We'd prefer braces be used at all times on conditionals - i.e. `if (themerTab) { ... } else { ... }`

Reply:
I think that's the [`curly`](https://eslint.org/docs/rules/curly) rule in ESLint.